packages/@jsii/go-runtime/jsii-runtime-go/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/aws/jsii-runtime-go
 
-go 1.18
+go 1.23.0
```

Comment:
This should be technically fine given the [lowest version still supported](https://endoflife.date/go) is 1.23.

But we might want to have a more formal announcement...TBD team discussion.

<img width="764" height="320" alt="Screenshot 2025-07-27 at 11 41 24 AM" src="https://github.com/user-attachments/assets/90115d3e-4056-4415-8e11-98a8654d4410" />